package.json

```diff
@@ -161,5 +161,8 @@
       "browser",
       "mocha"
     ]
+  },
+  "dependencies": {
```

Comment:
All the app dependencies should go in `/app/package.json`. This is only for dev dependencies. Also, we use exact working version instead of `^`.